app/views/shared/proposals/_tags_form.html.haml

```diff
@@ -1,13 +1,11 @@
 %fieldset
-  = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
-    .form-group
-      .tag-list
-        = f.label :review_tags
-        - tag_count = event.review_tags.count
-        - if tag_count > 0
+  - if event.review_tags.any?
+    = form_for proposal, url: reviewer_event_proposal_path(event, proposal), html: {role: 'form', remote: true} do |f|
+      .form-group
+        .tag-list
           = f.select :review_tags,
             options_for_select(event.review_tags, proposal.object.review_tags),
             {}, { class: 'multiselect review-tags', multiple: true }
           %button.pull-right.btn.btn-success{:type => "submit"} Update
-        - else
-          None
+  - else
+    None
```

Comment:
We definitely want to establish an event-wide check to see if tags are being used.  For now going with a method on event such as event.public_tags? should be good enough.  For now it can simply check to see if there are any proposal_tags set.  We may move to a checkbox or other flag that the event organizer would flip on to enable tagging.  That's why creating a method event.public_tags? is ideal since it is how the rest of the codebase would turn on the tagging UI portions.

For now, I'm fine using the same method, event.public_tags?, to turn off reviewer tags.  We may want to enhance reviewer tags to be free form but I don't see any event using it without having proposal tags too.